### ISSUE TITLE
Fails when running for n=65 nodes

### ISSUE THREAD

**Issue by adithyabhatkajake** [drand/drand#857]
When running drand for n=65 nodes, drand fails to perform DKG.

[Logs with scripts]
[drand-logs.zip](https://github.com/drand/drand/files/7576698/drand-logs.zip)

**Comment by willscott** [comment]
See if using the newer version of kyber works for you.

You can try this [PR](https://github.com/drand/drand/pull/856) against the most recent release, or take whatever version of drand you're trying and `go get github.com/drand/kyber@v1.1.7`, which will fix [this](https://github.com/drand/kyber/pull/23) issue.

**Comment by adithyabhatkajake** [comment]
I updated kyber on the master branch. I am still getting the same error. (Attached logs again
[drand-logs-after-updating-kyber.zip](https://github.com/drand/drand/files/7577214/drand-logs-after-updating-kyber.zip)
).

**Comment by adithyabhatkajake** [comment]
Here are the debug logs after running the code from `fix/log` branch

[logs-from-fixlog-branch.zip](https://github.com/drand/drand/files/7577258/logs-from-fixlog-branch.zip)

**Comment by nikkolasg** [comment]
Some of my findings...

**TLDR**: I didn't find anything suspicious in the logs (anything revealing a bug), the most likely reason the DKG is failing from the logs is because the timeout is too short here (10s). I I invite you to try with 1m or 2m even and send us back the logs.

Explanation:
* Nodes correctly sent all their deals at the beginning of the DKG (22:15:50)
```bash
cat *log | ack -i "echoBroadcast" | ack -i push | ack -v -i "response" | ack -i "2021-11-21T22:15:50" | wc -l
65
``` 

* Nodes moved to Response phase only because of the timeout (and not _before_ as it should happen in the optimistic case like this one because FastSync is enabled):
```bash
cat *log | ack -i "moving to response phase" | ack -i phaser | wc -l
65
```

* When nodes moved to Response phase, they only received a portion of their deals. Look at the lines from the command above (remove `wc -l`) - one example:
```
2021-11-21T22:16:00.180Z  DEBUG  core/drand_control.go:170  [dkg-proto-log phaser moving to response phase, got 26 deals]
```
-> most nodes only received about a third of all the deals.  Only two nodes received all the deals:
```bash
cat *log | ack -i "moving to response phase" | ack -i phaser | ack -i 65 | wc -l
2
```
* Even at the "DKG deadline" (timeout * 3 because there are 3 phases), most nodes still haven't received all their deals
```
cat logs-3.21.43.14.log | ack -i "echoBroadcast" | ack -i "received new packet" | ack -i "dkg.DealBundle"
```
Last Deal accepted was at `2021-11-21T22:16:20` and then after drand do not accept DKG messages (check with `cat logs-3.21.43.14.log | ack -i "no dkg running" | cut -f1 | head -n1`)

**Comment by nikkolasg** [comment]
Closed as experiments were running with wrong flags
